src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -0,0 +1,321 @@
+#define MAX_RELAY_PROTOCOL_LENGTH                 8U
+#define MAX_TLS_VERSION_LENGTH                    8U
+#define MAX_DTLS_CIPHER_LENGTH                    64U
+#define MAX_SRTP_CIPHER_LENGTH                    64U
+#define MAX_TLS_GROUP_LENGHTH                     32U
+#define MAX_PROTOCOL_LENGTH                       8U
+
+#define MAX_STRING_LENGTH                         255U
+
+/**
+ * @brief Type of Stats object requested by the application
+ * Reference https://www.w3.org/TR/webrtc-stats/#rtcstatstype-str*
+ */
+typedef enum {
+    CANDIDATE_PAIR, //!< ICE candidate pair statistics related to the RTCIceTransport objects
+    CERTIFICATE, //!< Information about a certificate used by an RTCIceTransport.
+    CODEC, //!< Stats for a codec that is currently being used by RTP streams being sent or received by this RTCPeerConnection object.
+    ICE_SERVER, //!< Information about the connection to an ICE server (e.g. STUN or TURN)
+    CSRC, //!< Stats for a contributing source (CSRC) that contributed to an inbound RTP stream.
+    DATA_CHANNEL, //!< Stats related to each RTCDataChannel id
+    INBOUND_RTP, //!< Statistics for an inbound RTP stream that is currently received with this RTCPeerConnection object
+    LOCAL_CANDIDATE, //!< ICE local candidate statistics related to the RTCIceTransport objects
+    OUTBOUND_RTP, //!< Statistics for an outbound RTP stream that is currently received with this RTCPeerConnection object
+    PEER_CONNECTION, //!< Statistics related to the RTCPeerConnection object.
+    RECEIVER, //!< Statistics related to a specific receiver and the corresponding media-level metrics
+    REMOTE_CANDIDATE, //!< ICE remote candidate statistics related to the RTCIceTransport objects
+    REMOTE_INBOUND_RTP, //!< Statistics for the remote endpoint's inbound RTP stream corresponding to an outbound stream that
+                        //!< is currently sent with this RTCPeerConnection object
+    REMOTE_OUTBOUND_RTP, //!< Statistics for the remote endpoint's outbound RTP stream corresponding to an inbound stream that
+                         //!< is currently sent with this RTCPeerConnection object
+    SENDER, //!< Statistics related to a specific RTCRtpSender and the corresponding media-level metrics
+    TRACK, //!< Statistics related to a specific MediaStreamTrack's attachment to an RTCRtpSender
+    TRANSPORT, //!< Transport statistics related to the RTCPeerConnection object.
+    SCTP_TRANSPORT, //!< SCTP transport statistics related to an RTCSctpTransport object
+    TRANSCEIVER, //!< Statistics related to a specific RTCRtpTransceiver
+    RTC_ALL //!< Report all supported stats
+} RTC_STATS_TYPE;
+
+/**
+ * @brief Indicates computing states in the checklist
+ * Reference: https://www.w3.org/TR/webrtc-stats/#rtcstatsicecandidatepairstate-enum
+ */
+typedef enum {
+    FROZEN,    //!< A check for the pair hasn't been performed because it is waiting for other checks to complete
```

Comment:
Same here. Think of how you can shorten the name of the enum. For example ICE_PAIR_STATE and then this entry will be ICE_PAIR_STATE_FROZEN

Reply:
Done